src/ensembl/stories/hooks/use-outside-click/useOutsideClick.stories.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState, useRef } from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import useOutsideClick from 'src/shared/hooks/useOutsideClick';
+
+import styles from './useOutsideClick.stories.scss';
+
+storiesOf('Components|Hooks/useOutsideClick', module).add('default', () => {
```

Comment:
> Components|Hooks/useOutsideClick

Hooks aren't components 😄 They probably deserve a section of their own.

Reply:
That's done